server/game/cards/10-TEL/KitsuMotso.js

```diff
@@ -0,0 +1,24 @@
+import { CardTypes, Players } from '../../Constants.js';
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl.js');
+
+class KitsuMotso extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Move a character in',
+            condition: context => context.game.isDuringConflict()
+                && context.source.isParticipating()
+                && context.player.hand.size() < context.player.opponent.hand.size(),
+            target: {
+                cardType: CardTypes.Character,
+                activePromptTitle: 'Choose a character to move in',
```

Comment:
we don't usually put `activePromptTitle` on abilities with one target... Is it necessary here?